applications/luci-app-packet-capture/root/usr/sbin/packet_capture_start

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+. /usr/share/libubox/jshn.sh
+
+json_load "$1"
+json_get_var interface interface
+json_get_var filter filter
+json_get_var duration duration
+json_get_var packets packets
+json_get_var verbose verbose
+json_get_var domains domains
+json_get_var file file
+
+args="-n"
+
+if [ "$domains" == "1" ];then
+	args=""
+fi
+
+if  [ -n "$interface" ];then
+	args="$args  -i  $interface"
```

Comment:
Check for `-e "/sys/class/net/$interface"` to prevent shell injection via crafted `interface` JSON field value